pkg/template/mixer/BUILD

```diff
@@ -0,0 +1,26 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
+load("@org_pubref_rules_protobuf//go:rules.bzl", "go_proto_library")
+
+go_proto_library(
```

Comment:
i believe we want to use `gogoslick_proto_library` here. Any reason why we are not?